### PR TITLE
[#60] refactor of release

### PR DESCRIPTION
**Pull request by pavertomato**
it was bad idea to use yaml syntax for scripts, seems like it parsed in different way for my cloud build and for yours.
i've moved decryption to script file
i'll update target branch as you create test branch

### CODE REVIEW

File: integration_test/scripts/publish-images.sh [hunk 1]
```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 # imageProject publishFlag repositoryString
```
Comment (danielbeaudreau) [inline integration_test/scripts/publish-images.sh:2]: please update this list for the final PR

Comment (pavertomato) [reply integration_test/scripts/publish-images.sh]: done

**Comment by pavertomato** [comment]
> I created a branch "debug"

updated

**Comment by pavertomato** [comment]
https://console.cloud.google.com/cloud-build/builds/6f017231-54cd-41d0-97ec-a65ee075e89d?authuser=1&project=gcp-healthcare-oss-test

some of gradle repo is down, hope it's temporary

**Comment by danielbeaudreau** [comment]
Ah, hopefully it resolves soon.
In the mean time Egor can you please create a rollback PR which undoes the original code change submitted today in master, so that the release can proceed as normal?

**Comment by pavertomato** [comment]
> Ah, hopefully it resolves soon.
> In the mean time Egor can you please create a rollback PR which undoes the original code change submitted today in master, so that the release can proceed as normal?

seems like now it's fine
https://console.cloud.google.com/cloud-build/builds/a01273da-8363-46ca-b870-bff4592a5da5?authuser=1&project=gcp-healthcare-oss-testand 
>rollback PR which undoes the original code change submitted today in master, so that the release can proceed as normal?
since release is created successfully, may be we'll merge debug branch into master to avoid duplication of commits? or i create same PR but without debug messages

**Comment by danielbeaudreau** [comment]
ok, can you please create a new PR without the debug print statements and I'll merge into master? thanks

**Comment by pavertomato** [comment]
https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/pull/67

**Pull request merged**